editors/code/src/config.ts

```diff
@@ -100,8 +100,12 @@ export class Config {
     get serverPath() {
         return this.get<null | string>("server.path") ?? this.get<null | string>("serverPath");
     }
-    get serverExtraEnv() {
-        return this.get<Env | null>("server.extraEnv") ?? {};
+    get serverExtraEnv(): Env {
+        const extraEnv =
+            this.get<{ [key: string]: string | number } | null>("server.extraEnv") ?? {};
+        return Object.fromEntries(
```

Comment:
I suppose there must be a way to only change the values that aren't strings, but it doesn't really matter here.